scripts/run53-multicap-probe.py

```
@@ -0,0 +1,160 @@
+        # generate-live-payment
+        payload = {
+            "orchestrator": base64.b64encode(info.SerializeToString()).decode("ascii"),
+            "type": "byoc",
+            "capabilities": base64.b64encode(byoc_caps.SerializeToString()).decode("ascii"),
+        }
+        req = urllib.request.Request(
+            f"{signer_origin}/generate-live-payment",
+            data=json.dumps(payload).encode(),
+            headers={"Content-Type": "application/json", "Livepeer-Capability": cap, **headers},
+            method="POST",
+        )
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)